applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -246,13 +253,14 @@ config BT_DEVICE_APPEARANCE
 config BT_ISO_BROADCASTER
 	default y
 
-config BT_AUDIO_BROADCAST_SOURCE
+config BT_BAP_BROADCAST_SOURCE
 	default y
 
 config BT_ISO_TX_BUF_COUNT
 	default 2
 
 config BT_AUDIO_BROADCAST_SRC_STREAM_COUNT
+	int "Number of broadcast source streams"
```

Comment:
Is there a specific reason for adding the text on this one (and for BT_AUDIO_BROADCAST_NAME)?
There are several other configs without text.
This is a .defaults file, meaning that the configs should be explained in the Kconfig file where they are defined